module/VuFind/src/VuFind/ILS/Driver/KohaILSDI.php

```diff
@@ -968,11 +964,12 @@ public function getHolding($id, array $patron = null)
                 'notes'        => $notes["MFHD"],
                 //'reserve'      => (null == $rowItem['RESERVES'])
                 //    ? 'N' : $rowItem['RESERVES'],
-                'reserve'      => 'N',
+                //'reserve'      => 'N',
+		'reserve'      => ($rowItem['NOTFORLOAN']) ? 'X':'N',
```

Comment:
Normally the 'reserve' field is used to track items held on course reserve (i.e. temporarily at a service desk to support students in a particular class) and the value is either 'Y' or 'N'. You are doing something non-standard here by setting this to 'X'. I would not recommend this approach, as it may cause confusion.